frappe/core/doctype/report/report.py

```diff
@@ -144,6 +144,20 @@ def create_report_py(self):
 			make_boilerplate("controller.py", self, {"name": self.name})
 			make_boilerplate("controller.js", self, {"name": self.name})
 
+	def translate_columns(self, columns):
+		translated_columns = []
+		for column in columns:
+			if isinstance(column, str):
+				translate = column.split(":", 1)
+				translate = '{0}:{1}'.format(_(translate[0]), translate[1])
```

Comment:
What's the split for? Tests are failing due to not all reports having this kind of structure.